network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/bootstrap_spacer_block.html

```diff
@@ -0,0 +1 @@
+<div class="mt-{{value.size}}"></div>
```

Comment:
this missing newline is intentional, and one of those times where github's insistence on it is plain wrong.